app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -12,6 +12,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
 
```

Comment:
This commit was mistakenly named as the last commit it is actually bolding the necessary texts